### ISSUE TITLE
Allow props forwarded to the svg component in typescript

### ISSUE THREAD

**Issue by zozoens31** [levrik/mdi-react#48]
Since we can forward props to the `svg` element (eg `style`), this should be reflected in the props type definition `MdiReactIconProps`.

To reproduce:
```javascript
import CloseIcon from 'mdi-react/CloseIcon'

const MyComponent: React.FC<{}> = () => {
  return <CloseIcon style={{marginLeft: 5}} />
}
```

This raises a type-script error 
```
error TS2322: Type '{ style: { marginLeft: number; }; }' is not assignable to type 'IntrinsicAttributes & MdiReactIconProps & { children?: ReactNode; }'.
  Property 'style' does not exist on type 'IntrinsicAttributes & MdiReactIconProps & { children?: ReactNode; }'.
```

**Comment by levrik** [comment]
You're right. I will take a look at #50 soon.

**Comment by levrik** [comment]
Released in `6.4.0`.
